lib/components/sidebar/active-link.tsx

```diff
@@ -20,42 +20,32 @@ const ActiveLink: React.FC<Props> = React.memo(({ href, text }) => {
   const isActive = pathname === href
 
   return (
-    <div className={`link ${isActive ? 'active' : ''}`}>
+    <>
       <Link href={href}>
-        <a className={isActive ? 'active' : ''}>
+        <a className={`link ${isActive ? 'active' : ''}`}>
           {title}
           {subtitle && <span>&nbsp;{subtitle}</span>}
         </a>
       </Link>
       <style jsx>{`
         .link {
-          width: 100%;
-          color: ${theme.palette.accents_5};
           display: flex;
-          height: 2.25rem;
-          align-items: center;
-          justify-content: flex-start;
-          cursor: pointer;
+          align-items: baseline;
+          font-size: 1rem;
+          color: ${theme.palette.accents_6};
+          padding: calc(${theme.layout.unit} * 0.375) 0;
+          transition: all 200ms ease;
         }
 
         a {
-          color: ${theme.palette.accents_7};
-          font-size: 1rem;
-          transition: all 200ms ease;
-          font-weight: 400;
-          display: inline-flex;
-          align-items: baseline;
+          font: inherit;
         }
 
-        a.active {
+        .link.active {
           color: ${theme.palette.success};
           font-weight: 600;
         }
 
-        a.active span {
-          color: ${theme.palette.successLight};
-        }
-
         span {
           font-size: 0.75rem;
           color: ${theme.palette.accents_4};
```

Comment:
The subheading is used in Chinese documents, it is a separate color and needs to be set active in the mobile view as well.

Reply:
@unix Thanks, fixed.